src/plugins/oSnap/components/TransactionBuilder/TransferFunds.vue

```diff
@@ -85,18 +85,18 @@ watch(selectedTokenAddress, updateTransaction);
       <span class="text-skin-text">{{ $t('safeSnap.asset') }}</span>
       <AvatarToken
         :address="
-          selectedToken.address === 'main'
+          selectedToken?.address === 'main'
             ? ETH_CONTRACT
-            : selectedToken.address
+            : selectedToken?.address
```

Comment:
`address` prop is `AvatarToken` is requried, is it possible to add a `if` condition to hide this if `selectedToken` is undefined?

Reply:
good call